website/docs/Helpers/validation.md

```diff
@@ -0,0 +1,182 @@
+---
+id: validation
+title: Validation Helpers
+sidebar_label: Validation Helpers
+---
+
+# Validation Helpers
+Here we have functions to validate the different datatypes and strings. Functions will return `boolean` values based on the validation.
+
+## Functions
+
+### is_bool()
+
+*core_lib.helpers.validation.is_bool()* [[source]](https://github.com/shay-te/core-lib/blob/5b8b2a4ca73dfd29138a216eb1f5648a5ae9be55/core_lib/helpers/validation.py#L10)
+
+Will validate if the passed argument is type `boolean` or not.
+
+```python
+def is_bool(st) -> bool:

```

Comment:
do you think we can rename parameter `st` to `val`. since it can be any value?
(docs and code)